getting-started/ts-example/.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
in general, why can't you use our `eslint.config.js` from main folder and extend from it ?

Reply:
That's what he did above https://github.com/open-telemetry/opentelemetry-js/pull/2297/files#diff-4bb0b44ed2e9b719ea2593f63cf9aebd302f9e7a1f3a109e5c35f6ee59d95d87R4

I think he probably just forgot to apply the suggestion to both eslintrc files.

Reply:
I was not able to use the project eslint config for typescript files because it requires that getting-started being added to the projects. I get this error when using the project eslint config:
```
0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: getting-started/ts-example/traced-example/tracing.ts.
The file must be included in at least one of the projects provided
```